lib/slack.ex

```diff
@@ -157,9 +165,94 @@ defmodule Slack do
     end
   end
 
+  @doc ~S"""
+  Turns a string like `"@USER_NAME"` into the ID that Slack understands (`"U…"`).
+  """
+  def lookup_user_id("@" <> user_name, slack) do
```

Comment:
> Thoughts on extracting these functions into their own module?

I did consider this, but couldn't really decide.  The biggest downside, to me, is that they're still based on the Slack data structure.  (P.S.  I do feel that should be converted into a struct, on an unrelated note.)  You have to pass that into them.

Should the `send_*()` function go into their own module?  Would it be the same module as the lookups (which they now use) or a different one?

I'm totally open to your thoughts on this.